webapp/channels/src/components/post_view/gm_conversion_message/gm_conversion_message.tsx

```diff
@@ -36,8 +36,9 @@ function GMConversionMessage(props: Props): JSX.Element {
         ),
     );
 
-    const convertedByUserUsername = userProfiles.find((user) => user.id === convertedByUserId)!.username;
-    const gmMembersUsernames = userProfiles.map((user) => renderUsername(user.username));
+    // we can memoize on profile length as the number of users in a GM always stay constant
```

Comment:
This comment is a bit misleading for me.
What remains constant is the number of users in the original GM (among other things because GMs cannot add/remove users and the GM exist no more). Therefore the message props will always have the same length for `gmMembersDuringConversionIDs` (that translate to always the same length for `userProfiles`).

Reply:
Removed.